tests/blur.py

```diff
@@ -66,3 +71,102 @@ def blur_fast_gaussian(cfg: ngl.SceneCfg):
         ),
     )
     return ngl.Group(children=(blur, ngl.DrawTexture(blurred_texture)))
+
+
+_N = 10
+
+
+@test_cuepoints(width=540, height=808, points=get_grid_points(_N, _N), keyframes=5, tolerance=5)
+@ngl.scene(controls=dict(show_dbg_points=ngl.scene.Bool()))
+def blur_hexagonal(cfg: ngl.SceneCfg, show_dbg_points=False):
+    mi = load_media("city")
+    cfg.aspect_ratio = (mi.width, mi.height)
+    cfg.duration = 5
+
+    source_texture = ngl.Texture2D(data_src=ngl.Media(filename=mi.filename))
+    blurred_texture = ngl.Texture2D()
+    blur = ngl.HexagonalBlur(
+        source=source_texture,
+        destination=blurred_texture,
+        amount=ngl.AnimatedFloat(
+            [
+                ngl.AnimKeyFrameFloat(0, 0),
+                ngl.AnimKeyFrameFloat(cfg.duration, 1),
+            ]
+        ),
+    )
+
+    group = ngl.Group(children=(blur, ngl.DrawTexture(blurred_texture)))
+    if show_dbg_points:
+        cuepoints = get_grid_points(_N, _N)
```

Comment:
you can compute `_CUEPOINTS = get_grid_points(10, 10)` once top level and reuse it as argument to `@test_cuepoints()` and `get_points_nodes()`

Reply:
Good Point, Thanks !